Task_2.py

```diff
@@ -0,0 +1,34 @@
+import os
+from source import get_data
+import csv
+from operator import itemgetter
+
+# 2) Task_2.py: to create file apps-stats.cvs containing information about
+# average rating (overall) of each application (asin) and number of voters.
+def avg_rating(main_dict):
+    asins = []
+    for item in main_dict.values():
+        asins.append(item['asin'])
+    asins_value = dict.fromkeys(set(asins), [0, 0])
+    for item in main_dict.values():
+        asins_value[item['asin']] = [asins_value.get(item['asin'])[0] + item['overall'], asins_value.get(item['asin'])[1]+1]
+    avg_overall = []
+    for key, value in asins_value.items():
+        avg_overall.append([key, asins_value.get(key)[0] / asins_value.get(key)[1], asins_value.get(key)[1]])
+    return avg_overall
+
+# create file apps-stats.cvs containing information above:
+def file_create(data):
```

Comment:
This function is used across tasks, let's move it to separate module and import. No need to duplicate code.